x-pack/plugins/ml/server/routes/apidoc_scripts/schema_extractor.ts

```diff
@@ -90,65 +90,77 @@ export function extractDocumentation(
     return members;
   }
 
-  function resolveTypeArgument(type: ts.Type): ts.SymbolTable | string {
-    // required to extract members
-    type.getProperty('type');
-
-    // @ts-ignores
-    let members = type.members;
+  /**
```

Comment:
It's worth bumping the version referenced in `apidoc.json` as part of this PR to `7.11.0` I think.

Reply:
Fixed in https://github.com/elastic/kibana/pull/82582/commits/3c94eac2f315b45ed68de2e58144725c60126cdd